apps/mobile/src/features/approver/components/inputs-outputs-card.tsx

```diff
@@ -31,48 +31,52 @@ export function InputsAndOutputsCard({ inputs, outputs }: InputsAndOutputsCardPr
 
   const outputsWithBalance = outputs.map(addBalance);
   return (
-    <>
-      <Text variant="label01">
+    <Box gap="5">
+      <Text variant="label02">
         {t({
           id: 'approver.inputs-outputs.title',
           message: 'Inputs and Outputs',
         })}
       </Text>
-      <Text variant="label01">
-        {t({
-          id: 'approver.inputs-outputs.input.title',
-          message: 'Input',
-        })}
-      </Text>
-      <Box gap="4">
-        {inputsWithBalance.map(input => (
-          <UtxoRow
-            key={input.txid + input.address + input.btcBalance.amount}
-            txid={input.txid}
-            address={input.address}
-            btcBalance={input.btcBalance}
-            usdBalance={input.usdBalance}
-            isLocked
-          />
-        ))}
+      <Box gap="1">
+        <Text variant="label02">
+          {t({
+            id: 'approver.inputs-outputs.input.title',
+            message: 'Input',
+          })}
+        </Text>
+        <Box mx="-5">
+          {inputsWithBalance.map(input => (
+            <UtxoRow
+              key={input.txid + input.address + input.btcBalance.amount}
+              txid={input.txid}
+              address={input.address}
+              btcBalance={input.btcBalance}
+              usdBalance={input.usdBalance}
+              isLocked
+            />
+          ))}
+        </Box>
       </Box>
-      <Text variant="label01">
-        {t({
-          id: 'approver.inputs-outputs.output.title',
-          message: 'Output',
-        })}
-      </Text>
-      <Box gap="4">
-        {outputsWithBalance.map(output => (
-          <UtxoRow
-            key={output.address + output.btcBalance.amount}
-            address={output.address}
-            btcBalance={output.btcBalance}
-            usdBalance={output.usdBalance}
-            isLocked
-          />
-        ))}
+      <Box gap="1">
+        <Text variant="label02">
+          {t({
+            id: 'approver.inputs-outputs.output.title',
+            message: 'Output',
+          })}
+        </Text>
+        <Box mx="-5">
```

Comment:
Are all these `mx="-5"` to override the child? Can we avoid that? 🤔

Reply:
Good question. Many (if not all) of our layouts currently have a horizontal padding applied by default. This is a bit too opinionated and is defined in way too many places for me to be able to address within this PR. Cells and few other components on the other hand are full-width–something that was missed during initial implementations.
I'm hoping to address this at some point with adding few layout primitives, but this isn't a trivial task unfortunately.

Reply:
Thanks! Yes it seems its a SMELL of an underlying issue we should solve but agreed too much for this PR.